BRAINSConstellationDetector/src/landmarksConstellationCommon.h

```diff
@@ -150,9 +150,9 @@ ComputeMSP(SImageType::Pointer           input_image,
            double &                      cc);
 
 extern SImageType::Pointer
-CreatedebugPlaneImage(SImageType::Pointer               referenceImage,
-                      const RigidTransformType::Pointer MSPTransform,
-                      const std::string &               debugfilename);
+CreatedebugPlaneImage(SImageType::Pointer                 referenceImage,
```

Comment:
(repeat) Should referenceImage be passed by reference?

Reply:
Same as above -- _Yes, it is covered in PR#501. This PR only converted constant pass-by-value parameters to constant pass-by-reference parameters. The other PR converts non-constant pass-by-value parameters to constant pass-by-reference parameters. The changes are quite similar, so it likely could have been one PR, but they are a bit different so I separated them._